components/MDX.js

```diff
@@ -60,9 +60,8 @@ export function getMDXComponents() {
     th: props => <th {...props} className={`${props.className || ''} px-6 py-3 border-b border-gray-200 bg-gray-100 text-left text-xs leading-4 font-medium font-body text-gray-900 uppercase tracking-wider`} />,
     tr: props => <tr {...props} className={`${props.className || ''} bg-white`} />,
     td: props => <td {...props} className={`${props.className || ''} px-6 py-4 border-b border-gray-200 text-sm leading-5 text-gray-700 tracking-tight`} />,
-    pre: props => <div {...props} className={`${props.className || ''} my-8`} />,
-    inlineCode: props => <code {...props} className={`${props.className || ''} px-1 py-0.5 bg-gray-200 text-gray-800 rounded font-mono text-sm`} />,
-    code: CodeComponent,
```

Comment:
You can't remove the `CodeComponent` here because it will then break the properties of three backticks (code), as shown here:

![image](https://github.com/asyncapi/website/assets/76521428/117ccfb0-71d9-4673-b541-a1e4e6acac3a)

Reply:
I think PreComponent that i wrote handles the case. Can you tell where is this happening so that I can check again?

Reply:
I was trying to check if children type inside pre is code tag only. But the props are different in documentation. So, now I decided to just replace `CodeComponent` for `pre`.